corehq/apps/reports/display.py

```diff
@@ -73,59 +73,41 @@ def __init__(self, domain, xmlns, app_id=None):
         if app_id:
             self.app_id = app_id
         else:
+            form = get_form_analytics_metadata(domain, xmlns, app_id)
             try:
-                form = get_form_analytics_metadata(domain, xmlns, app_id)
                 self.app_id = form['app']['id']
-            except Exception:
+            except KeyError:
                 self.app_id = {}
 
-    @property
-    @memoized
-    def _metadata(self):
-        try:
-            return get_form_analytics_metadata(self.domain, self.xmlns, self.app_id)
-        except Exception, e:
-            _assert = soft_assert('{}@{}'.format('brudolph', 'dimagi.com'), notify_admins=True)
-            _assert(False, 'Failed on domain: {} | xmlns: {} | app_id: {} | error: {}'.format(
```

Comment:
Ah, it's because of quickcache, which I recently added to `get_form_analytics_metadata` and only accepts hashable arguments: `Bad type "<type 'dict'>": {}`